packages/cascara/src/specs/Button/Button.Spec.mdx

```diff
@@ -0,0 +1,49 @@
+---
+title: Button
+status: Draft
+
+---
+
+# Button
+
+A button is an atom level element that is used alone as well in a variety of larger components.
+
+## Existing Button Types
+
+- Priority/Intention (not clearly defined, but implied)
+	- Primary
+	- Secondary
+	- Tertiary
+	- Text only
+	- Destructive
+- Icon
+	- With icon
+	- Icon only
+- Boolean button
```

Comment:
It seems that the Boolean button could be converted into a checkbox that is displayed as a switch, visually it is more descriptive. What do you think @CaffeinatedAllie and @brianespinosa?

Reply:
We already have buttons that also act as a toggle.... think the UI element where the user picks the day of the week. An on/off switch would be awkward for that kind of UI element.